Testing/CASLTests/Exceptions/UnknownPlatformExceptionTests.cs

```diff
@@ -21,31 +22,35 @@ public void Ctor_WhenInvokedWithNoParam_CorrectlySetsMessage()
         var exception = new UnknownPlatformException();
 
         // Assert
-        Assert.Equal("The platform is unknown.", exception.Message);
+        var expectedExceptionMessage = "The platform is unknown.";
+        exception.Message.Should().Be(expectedExceptionMessage);
     }
 
     [Fact]
     public void Ctor_WhenInvokedWithSingleMessageParam_CorrectlySetsMesage()
     {
         // Act
-        var exception = new UnknownPlatformException("test-message");
+        var exceptionMessage = "test-message";
+        var exception = new UnknownPlatformException(exceptionMessage);
 
         // Assert
-        Assert.Equal("test-message", exception.Message);
+        exception.Message.Should().Be(exceptionMessage);
     }
```

Comment:
Please refactor to the suggested change shown below.

Reason: This will make the test follow the AAA (triple A) test layout.

```suggestion
    [Fact]
    public void Ctor_WhenInvokedWithSingleMessageParam_CorrectlySetsMesage()
    {
        // Arrange
        var expected = "test-message";
        
        // Act
        var exception = new UnknownPlatformException(expected);

        // Assert
        exception.Message.Should().Be(expected);
    }
```

Reply:
Refactored test to follow the triple A test layout.